### DISCUSSION TITLE
Purpose Of Start-As-Admin?/以管理员身份启动的目的?

### DISCUSSION THREAD

**Discussion by TJCrinc** [JasonWei512/EnergyStarX#37] [Q&A] [answered]
Ill post in both english and chinese (using google translate).

I am curious what the purpose of starting with Windows as Admin.
-------------------
我会用英文和中文发帖（使用谷歌翻译).

我很好奇以管理员身份启动 Windows 的目的是什么.

**Comment by JasonWei512** [comment, marked as answer]
Energy Star X cannot throttle programs with higher privileges.
For example, if `notepad` is running as admin, and Energy Star X is not running as admin, then `notepad` cannot be throttled by Energy Star X.
